src/raftstore/store/debug.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::{error, result};
+use kvproto::debugpb::*;
+use rocksdb::DB;
+
+use raftstore::store::keys;
+use raftstore::store::Engines;
+use raftstore::store::engine::Peekable;
+use storage::{CF_DEFAULT, CF_LOCK, CF_RAFT, CF_WRITE};
+
+pub type Result<T> = result::Result<T, Error>;
+
+quick_error!{
+    #[derive(Debug)]
+    pub enum Error {
+        InvalidArgument(msg: String) {
```

Comment:
Could we use `&'static str` instead of `String`?

Reply:
I think `String` is more flexible and Debug service is not performance-critical.